assets/src/dashboard/components/multiPartPill/index.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const PillContainer = styled.div`
+  display: inline-flex;
+  align-items: center;
+  height: 44px;
+  padding: 0 10px;
+  border-radius: ${({ theme }) => theme.border.buttonRadius};
+  box-shadow: 0px 2px 11px rgba(0, 0, 0, 0.25);
+`;
+
+const PillSeparator = styled.div`
+  width: 1px;
+  height: 20px;
+  margin: 0 10px;
+  background-color: ${({ theme }) => theme.colors.gray75};
+`;
```

Comment:
you may be able to get away with making this a wrapper and making this a `border-right` or something. May or may no help 🤷

Reply:
Yeah, I updated it to a wrapper and used the `:before` content to make the lines nice and goodly for every child after the first .